senders/victorops/send.go

```diff
@@ -0,0 +1,130 @@
+package victorops
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/writeas/go-strip-markdown"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/senders/victorops/api"
+)
+
+// SendEvents implements Sender interface Send
+func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plot []byte, throttled bool) error {
+	createAlertRequest := sender.buildCreateAlertRequest(events, trigger, throttled, plot, time.Now().Unix())
+	err := sender.client.CreateAlert(contact.Value, createAlertRequest)
+	if err != nil {
+		return fmt.Errorf("error while sending alert to victorops: %s", err)
+	}
+	return nil
+}
+
+func (sender *Sender) buildCreateAlertRequest(events moira.NotificationEvents, trigger moira.TriggerData, throttled bool, plot []byte, time int64) api.CreateAlertRequest {
+
+	var messageType api.MessageType
```

Comment:
It seems we have problems with determining the type of message here and in PagerDuty.
See how it is done in [pushover](https://github.com/moira-alert/moira/blob/master/senders/pushover/pushover.go#L57)
But I think that this decision is unnecessary. It seems the easiest way to use a ```GetSubjectStat``` here and use this state in calculating the correct message type. Let's redo it here and in PagerDuty